challenges/fizzBuzzTree/fizz-buzz-tree.js

```diff
@@ -0,0 +1,16 @@
+const fizzBuzzTree = (tree) => {
```

Comment:
this function doesn't take the tree as an argument, it takes a value in the tree. Can you change this so that you can run the function once for the whole tree?